db/migrate/20221129163510_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
- [ ]  Kindly change user to author so as to generate author_id i.e `t.references :author, foreign_key: { to_table: 'users' } `